cwms-data-api/src/main/java/cwms/cda/data/dao/location/kind/OutletDao.java

```diff
@@ -179,7 +180,7 @@ public void deleteVirtualOutlet(String officeId, String projectId, String virtua
     private Outlet mapToOutlet(PROJECT_STRUCTURE_OBJ_T projectStructure,
```

Comment:
The projectStructure here is the outlet, so name it outlet

Reply:
Zack updated it to outlet here
https://github.com/USACE/cwms-data-api/blob/develop/cwms-data-api/src/main/java/cwms/cda/data/dao/location/kind/OutletDao.java#L199